site/gdocs/components/centered-article.scss

```diff
@@ -195,6 +195,62 @@ $banner-height: 200px;
     margin-bottom: 32px;
 }
 
+.deprecation-notice {
+    --bg-color: #fff5d8;
```

Comment:
![image](https://github.com/user-attachments/assets/a215efc3-c50f-4390-8d3e-f5d9faa711cd)

need to zindex this higher, check out `variables.scss` 🙂

Reply:
The stickiness doesn't seem to work quite right when I do screen emulation. Not sure if that'll be an actual issue on mobile.

Reply:
![image](https://github.com/user-attachments/assets/5a61b682-d899-4bdf-9591-09b2c1d4cb53)

I don't think there shouldn't be yellow space between the header and the body on small when the notice is present 🙂

Reply:
Thanks, I didn't know about the z-index variables. I added a new one for the deprecation notice.

I can't reproduce the issue with stickiness on with screen emulation.

The yellow space between the header and the body I also can't reproduce. Maybe it's because of the missing date (don't know why it would be missing)?

Reply:
Ah, now I reproduced the stickiness issue with small screen. Looking into it.

Reply:
Pushed a fix for that.

Reply:
And yup, looks like it's only an issue when there's no publishedAt date, so no worries there 🙂